File: src/main/java/ru/sbp/trm/core/BotActions.java
package ru.sbp.trm.core;

import lombok.Getter;

@Getter
public enum BotActions {
    SHOW_MAIN_MENU("Чего изволите?","Основное меню"),
    SHOW_FREE("Какой стенд хотите забронировать?", "Свободные стенды"),
    SHOW_MY("Какой стенд желаете освободить?", "Мои стенды"),
    ORDER("Резервирование стенда", ""),
    DISMISS("Освободить стенд", "");
    private final String messageText;
    private final String buttonText;

    BotActions(String messageText, String buttonText) {
        this.messageText = messageText;
        this.buttonText = buttonText;
    }

}

File: src/main/kotlin/trm/core/BotActions.kt
package trm.core

enum class BotActions(val messageText: String) {
    SHOW_MAIN_MENU("Чего изволите?"),
    SHOW_FREE("Какой стенд хотите забронировать?"),
    SHOW_MY("Какой стенд желаете освободить?"),
    ORDER("Резервирование стенда"),
    DISMISS("Освободить стенд");
}